kube/charts/kubernetes/context.go

```diff
@@ -22,11 +22,69 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/docker/compose-cli/api/context/store"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
 )
 
 // ListAvailableKubeConfigContexts list kube contexts
 func ListAvailableKubeConfigContexts(kubeconfig string) ([]string, error) {
+	config, err := getKubeConfig(kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+	contexts := []string{}
+	for k := range config.Contexts {
+		contexts = append(contexts, k)
+	}
+	return contexts, nil
+}
+
+// LoadConfig returns kubeconfig data referenced in the docker context
+func LoadConfig(ctx store.KubeContext) (*genericclioptions.ConfigFlags, error) {
+	if ctx.FromEnvironment {
+		return nil, nil
+	}
+	config, err := getKubeConfig(ctx.KubeconfigPath)
+	if err != nil {
+		return nil, err
+	}
+	contextName := ctx.ContextName
```

Comment:
We should rename this (maybe "KubeContextName") to avoid confusion with the name of the Docker context